Sources/StreamVideo/OpenApi/generated/Models/QueryCallStatsRequest.swift

```diff
@@ -1,45 +1,52 @@
 //
-// QueryCallStatsRequest.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct QueryCallStatsRequest: Codable, JSONEncodable, Hashable {
+    
+public final class QueryCallStatsRequest: @unchecked Sendable, Codable, JSONEncodable, Hashable {
+    
     public var filterConditions: [String: RawJSON]?
     public var limit: Int?
     public var next: String?
     public var prev: String?
-    public var sort: [SortParamRequest]?
-
-    public init(filterConditions: [String: RawJSON]? = nil, limit: Int? = nil, next: String? = nil, prev: String? = nil, sort: [SortParamRequest]? = nil) {
+    public var sort: [SortParamRequest?]?
+
+    public init(
+        filterConditions: [String: RawJSON]? = nil,
+        limit: Int? = nil,
+        next: String? = nil,
+        prev: String? = nil,
+        sort: [SortParamRequest?]? = nil
```

Comment:
with this change, you can also do `[nil]`, which doesn't feel right.

Reply:
Good catch. Fixed.